serde/src/lib.rs

```diff
@@ -70,20 +108,30 @@ pub const fn uint_max_size(max_value: u64) -> usize {
     };
     ((bits + 6) / 7) as usize
 }
+
 /// Maximum size in bytes for an `Option<T>` value in binary serialization.
 ///
 /// `inner_size` is the maximum size of its inner value `T`.
 pub const fn option_max_size(inner_size: usize) -> usize {
     1 + inner_size
 }
+
 /// Maximum size in bytes for a `Vec<T>` value in binary serialization.
 ///
 /// `inner_size` is the maximum size of its inner value `T`. `max_elems` is the maximum number of
 /// elements in that `Vec<T>`.
-pub const fn vec_max_size(inner_size: usize, max_elems: usize) -> usize {
+pub const fn seq_max_size(inner_size: usize, max_elems: usize) -> usize {
     uint_max_size(max_elems as u64) + inner_size * max_elems
 }
 
+pub const fn max(a: usize, b: usize) -> usize {
```

Comment:
Why are we not using `std::cmp::max`?

Reply:
Because `std::cmp::max` is not a `const fn`.